admin-dev/themes/new-theme/js/pages/product/services/images.js

```diff
@@ -62,17 +62,35 @@ export const replaceImage = async (selectedFile, newFile, formName, token) => {
   const formData = new FormData();
   formData.append(`${formName}[file]`, newFile);
   formData.append(`${formName}[_token]`, token);
+  formData.append('_method', 'PATCH');
 
   return $.ajax(replaceUrl, {
     method: 'POST',
     data: formData,
-    contentType: false,
+    cache: false,
```

Comment:
```suggestion
```

Documentation :
If set to false, it will force requested pages not to be cached by the browser. Note: Setting cache to false will only work correctly with HEAD and GET requests. It works by appending "_={timestamp}" to the GET parameters. The parameter is not needed for other types of requests, except in IE8 when a POST is made to a URL that has already been requested by a GET.